_posts/2021-07-28-how-to-update-to-cis-14.adoc

```diff
@@ -0,0 +1,156 @@
+---
+title: How to update to CIS AWS Foundations Benchmark v1.4.0
+categories: Upgrades
+image: /assets/img/guides/refresh_icon.png
+excerpt: Learn how to update to version 1.4.0 of the CIS AWS Foundations Benchmark
+tags: ["aws", "security", "compliance"]
+cloud: ["aws"]
+redirect_from: /static/guides/upgrades/how-to-update-to-cis-14/
+---
+:page-type: guide
+:page-layout: post
+
+:toc:
+:toc-placement!:
+
+// GitHub specific settings. See https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74 for details.
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+toc::[]
+endif::[]
+
+== Intro
+
+This guide will walk you through the process of updating version 1.3.0 to version 1.4.0 of the CIS AWS Foundations Benchmark.
+If your infrastructure is already compliant with v1.3.0, and you are looking to upgrade to v1.4.0,
+then this guide is for you. Alternatively, if you are starting from scratch, check out our
+https://gruntwork.io/guides/compliance/how-to-achieve-cis-benchmark-compliance/[How to achieve compliance with the CIS AWS Foundations Benchmark]
+guide instead.
+
+=== Previous versions of this guide
+- https://gruntwork.io/guides/upgrades/how-to-update-to-cis-13/[How to update to CIS AWS version 1.3.0]
+
+=== What you'll learn in this guide
+
+This guide consists of two main sections:
+
+<<core_concepts>>::
+An overview of the CIS AWS Foundations Benchmark v1.4.0 and why it is important to update your code for compatibility.
+
+<<deployment_walkthrough>>::
+The steps you need to take to update your code to be compliant with v1.4.0. It includes a
+<<compatibility_table,version compatibility table>> you can use as a reference to know which Gruntwork Repo version
+tag is compatible along with the manual steps you need to perform to achieve compliance with version v1.4.0 of the CIS
+AWS Foundations Benchmark.
+
+
+[[core_concepts]]
+== Core Concepts
+
+=== Background
```

Comment:
Would be nice to have a small paragraph here similar to the old guide: https://gruntwork.io/guides/upgrades/how-to-update-to-cis-13/#background.

Reply:
Added https://github.com/gruntwork-io/gruntwork-io.github.io/pull/512/commits/f212a029e3c9e959ca0fe77b2f321bf48f88f907

Reply:
Looks good!